### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by jmbowman**
Copied the existing theme from edx-documentation and added some packaging along the lines of cookiecutter-django-app.  Tests are still minimal and the full benefits of the theme aren't yet enumerated in the README (largely because I haven't looked through all the templates yet to figure out what they are).  Two minor changes that were made to the theme when copying it:

* `feedback_form_url` is now populated by an HTML rendering context event hook, instead of adding a callable to the default context only if we don't seem to be using the JSON builder.
* Switched the CDN link for modernizr to always use HTTPS, because the `//` prefix resolved to `file://` and failed to load when viewing the output locally.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+Even though they were written with ``edx-platform`` in mind, the guidelines
+should be followed for Open edX code in general.
+
+Reporting Security Issues
+-------------------------
+
+Please do not report security issues in public. Please email security@edx.org.
+
+Mailing List and IRC Channel
```
Comment (nedbat) [inline README.rst]: We have to stop mentioning the IRC channel.  We need better boilerplate for this.

File: docs/make.bat [hunk 2]
```diff
@@ -0,0 +1,281 @@
+@ECHO OFF
```
Comment (nedbat) [inline docs/make.bat]: Surely we don't need a make.bat!

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    long_description=README + '\n\n' + CHANGELOG,
+    author='edX',
+    author_email='oscm@edx.org',
+    url='https://github.com/edx/edx-sphinx-theme',
+    packages=[
+        'edx_theme',
+    ],
+    include_package_data=True,
+    install_requires=['six', 'Sphinx'],
+    license="AGPL 3.0",
```
Comment (nedbat) [inline setup.py]: This should be Apache 2.0

**Comment by lamagnifica** [comment]
👍 :)

**Pull request merged**